app/controllers/coronavirus_form/expert_advice_type_controller.rb

```diff
@@ -15,6 +15,8 @@ def submit
         values: @form_responses[:expert_advice_type],
         allowed_values: I18n.t("coronavirus_form.questions.#{controller_name}.options").map { |_, item| item.dig(:label) },
         other: @form_responses[:expert_advice_type_other],
+        exclusive: selected_exclusive?,
+        exclusive_values: I18n.t("coronavirus_form.questions.#{controller_name}.options").map { |_, item| item.dig(:label) if item.dig(:exclusive) == true },
```

Comment:
It looks like these `exclusive_values` are the same as the value used to determine `selected_exclusive?`.

Can this be DRY-ed up, so it obvious we're looking at the same thing? Perhaps move finding the `exclusive_values` into method, and have `selected_exclusive?` called `exclusive_values.first` ?

Reply:
Good suggestion, @leenagupte. I updated the code now. We only use them in 2 places atm, I'm happy to create a helper if we'll need them elsewhere.